assets/js/admin/course-settings-plugin-sidebar.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import { applyFilters } from '@wordpress/hooks';
+import {
+	PluginDocumentSettingPanel,
+	PluginSidebar,
+	PluginSidebarMoreMenuItem,
+} from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { dispatch, useSelect } from '@wordpress/data';
+import { Slot } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import CoursePricingPromoSidebar from './course-pricing-promo-sidebar';
+import CourseAccessPeriodPromoSidebar from './course-access-period-promo-sidebar';
+import CourseTheme from './course-theme';
+import CourseVideoSidebar from './course-video-sidebar';
+import SenseiIcon from '../../icons/logo-tree.svg';
+
+export const pluginSidebarHandle = 'sensei-lms-course-settings-sidebar';
+export const pluginDocumentHandle = 'sensei-lms-document-settings-sidebar';
+
+export const CourseSidebar = () => {
+	const hideCoursePricing = applyFilters( 'senseiCoursePricingHide', false );
+	const hideAccessPeriod = applyFilters(
+		'senseiCourseAccessPeriodHide',
+		false
+	);
```

Comment:
Should we add the hooks docs?

Reply:
Good point, added a new issue https://github.com/Automattic/sensei/issues/6071